### PR TITLE
Don't make `flutter drive` gate iOS devices.

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by chinmaygarde** [comment]
cc @yjbanov: I was able to get drive working on devices after this and a previous change. LMK if there is anything else to be done to enable the same on bots.

File: packages/flutter_tools/lib/src/commands/drive.dart [hunk 1]
```diff
@@ -205,56 +205,57 @@ void restoreTargetDeviceFinder() {
...
-    // we look for an Android device. If there's one, we use that. Otherwise,
-    // we launch a new iOS Simulator.
-    Device reusableDevice = devices.firstWhere(
-      (Device d) => d.isLocalEmulator,
-      orElse: () {
-        return devices.firstWhere((Device d) => d is AndroidDevice,
-            orElse: () => null);
-      }
-    );
+  if (os.isWindows) {
```
Comment (yjbanov) [inline packages/flutter_tools/lib/src/commands/drive.dart]: I don't think this check is needed, given the checks below that you added.

File: packages/flutter_tools/lib/src/commands/drive.dart [hunk 2]
```diff
@@ -205,56 +205,57 @@ void restoreTargetDeviceFinder() {
...
-    return devices.first;
-  } else if (os.isWindows) {
-    printError('Windows is not yet supported.');
-    return null;
-  } else {
-    printError('The operating system on this computer is not supported.');
-    return null;
   }
+
+  // If multiple devices are connected, inform the user that maybe she should
```
Comment (yjbanov) [inline packages/flutter_tools/lib/src/commands/drive.dart]: nit: s/she/they/

**Comment by yjbanov** [comment]
LGTM, but not sure we should submit until the waterfall clears up.

**Comment by yjbanov** [comment]
> LMK if there is anything else to be done to enable the same on bots

I'll do the dashboard change myself. As for the waterfall bots, we don't have any devices on those, so this change should not affect them.

**Pull request closed without merging**